### DISCUSSION TITLE
⚔️ Faction Rank Updates [PROPOSAL]

### DISCUSSION THREAD

**Discussion by adamhannigan** [sunflower-land/sunflower-land#4288] [Ideas]
The purpose of this proposal is to discuss the new faction rank requirements.

**Challenge**

Since faction launch, we had little control to which factions would become most popular or dominant. We stated that we would readjust rank requirements after launch to balance it with changing conditions in the game.

We initially based ranks on the percentage tiers in a faction. For instance, the top 1% had the top tier. At the time we hardcoded these values, meaning they did not dynamically change when emblem ownership changed.

This proposal is to readjust the numbers to reflect the current rankings.

**Release** 🚀 

We plan to roll out these changes on the 9th of September.

## Solutions

We have multiple solutions for the ranking system that we will discuss here.

### Option A

Continue with our **current pattern** where the top X% determine your faction rank. Based on current emblem ownership would work out to be the following:

![faction_ranks](https://github.com/user-attachments/assets/6a5dae9c-5bf8-4a3a-ae7c-6d0a503e4a75)

_These values are from 2 weeks ago. They would be updated to reflect % ownership on 2nd of September_

**Pros**

- Keeps the rank system consistent
- Encourages % ownership in a faction

**Cons**

- It may be cheaper for members in a different faction to reach top tier.

### Option B

Use a top X members approach that is consistent across all factions. For example:

- Top 50 members will be top rank
- Top 250 members will be second rank
- Top 500 members will be third rank

_Numbers above are placeholders. We would try work out roughly the same numbers on average in each rank we currently have._

**Pros**

- All factions have the same amount of members in each rank. This means fairer weekly competition as rank multipliers are even across factions.
- Simpler system that is standard across each faction

**Cons**

- Moves away from percentage approach
- Factions with cheaper emblems will find it cheaper to be top tier

### Option C 

Base the ranks on the price of emblems. For instance:

- Rank 1 = 10,000 SFL worth of emblems
- Rank 2 = 5,000 SFL worth of emblems
- Rank 3 = 2,000 SFL worth of emblems

**Pros**

- Each faction needs to pay the same amount of SFL to achieve a ranking
- Could lead to better self-balancing of faction members

**Cons**

- Cheaper factions will have more Rank 1 members.

## Conclusion

We would love to hear your thoughts on the Faction rank system options or if you have any alternatives. Please let us know in the comments below.

The plan is to release these updates on the **9th of September.** 🚀 

_Note: We will continue to readjust faction ranks every 2-3 months to keep things fresh and aligned in game._

**Comment by brian-man** [comment]
I don't understand option C.  Price based on what?  P2P price?  If I buy (or sell) large numbers of emblems, I directly impact player ranks?

**Reply by adamhannigan** [reply]
That option would be based on P2P price. These values wouldn't update instantly, they would readjust after 2- months

**Reply by Polysemouse** [reply]
The prices of P2P can be unstable.  How would the system determine the price of the emblems at the snapshot?

**Reply by adamhannigan** [reply]
That is true. In that approach some sort of average would need to be considered. It all sounds quite complicated

**Comment by snorri-snorri** [comment]
Below is from Luksm, can't access his GitHub 



Here is my proposal :

1.	Each week you count the number of members in each faction that have participated at least once each to kitchen deliveries. 
Not pet feeding (might lead to overfeeding just for that) neither factions chores (which are purely a personal gain for mark). 
Burn in the kitchen deliveries seems to be the core of the leaderboard competition, and so the faction's competition too.


2.	Then you get the total amount of mark generated through the kitchen deliveries that week in that faction. And you divide 2. by 1.


3.	The "bonus faction +10% xp" is only available to players who have contributed at least one of each kitchen deliveries during the past week (same code as for pet, right?)


4.	This way, that's the MEDIAN investment of the collective that is used as a way to rank factions. 4 "archetypes" possible

A. small group of active elite, rest quite passive : generate an high score, yet encourage passive player to seek that "easy bonus" (carried) next week. 
But then drop the score if they don't invest themselves decently for their own rank

B. No elite, rest still passive : Well, losing strategy, oviously, but then weekly reward for top 50 should turn that configuration into A. next week

C. No real elite, rest quite active : Ideal collective behavior. Quite utopist :D 
Yet, would lead to median score quite high (everyone burning accordigly to their rank's expectations). 
The most efficient way to play collectively. Yet, as soon as personal greed is triggered by the weekly rewards for top 50, then it will turn into D.

D. Small group of elite, rest of player quite active : The perfect faction. 
Top players inside compete between themselves in order to get weekly rewards, rest is collectively invested in order to get the top faction mark bonus (marks utility is at the top in players's mind, here). Then that's the emblem's price that will regulate and break this : 
emblems will naturally rise leading to collective crumbling (players active getting out, players leeching going in)

With this proposal, each week, players can use faction in three different ways :

â€¢	chores for personal marks gain, equal for everyone

â€¢	pet feeding for collective uncompetitive goal, in order to gain a collective xp boost (faction's collective spirit and cohesion)

â€¢	kitchen deliveries : "war effort" (feeding the troops). Competitive INSIDE the faction in order to differientate leaderbaord ranks. 
Competitive between factions in order to determine the top 1 faction.

Of course, such proposal goes with "Option A"

You also have (imo, probably wrong) a self regulating faction system, using personal gain (top 50), collective gain (+10% mark bonus) and emblems' marketprice as a three ways custodians of each other.
And that should maximize burn, since it should encourage more kitchen burn overtime and higher emblems prices then fees over time

**Comment by Alucard0389** [comment]
Option A, the percentage-based rank determination, offers several advantages that make it a strong choice:

Advantages of Option A:

1.	Consistency and Familiarity:
This option preserves the existing system that players are already accustomed to. It’s easier for players to adapt to changes when the fundamental structure remains the same.
	
2.	Encourages Active Participation:
Since ranks are based on the percentage of emblem ownership within a faction, this system motivates players to acquire more emblems and improve their standing within the faction. It promotes engagement and investment in the faction.
	
3.	Relative Fairness Within Factions:
The ranking is determined relative to other members within each faction. This means that players, regardless of their faction’s size, have opportunities to rise in the ranks as long as they are active in faction activities.
	
4.	Dynamic Adjustment:
Although the values were initially set at launch, this system can be updated regularly to reflect current conditions, leading to a dynamic and responsive ranking structure that adjusts to the evolving state of factions.

Why Choose Option A:

If consistency and a system that promotes engagement within a faction are important, Option A is the best choice. It allows players to measure their progress relative to their faction peers and ensures that the existing system is not abruptly changed, making the adjustments easier to accept. Additionally, it maintains healthy competition within factions, leading to a more intense and exciting gameplay experience.

But let's see what other people say❤️

**Comment by brian-man** [comment]
The root issue, IMO, is the imbalance from the beginning -- that the volume of emblems per faction are different and the average cost to obtain them wasn't consistent either.  Ignoring the latter for a moment....

The first step in balancing the factions is coming up with a plan such that all factions have the same amount of emblems "available".

Even that is hard because once you balance the emblems, if a farm is banned or lost then all the emblems are effectively lost and the imbalance returns.

**Comment by raulmonoanivia** [comment]
A is definitely the one that makes more sense. It allows to consistently climb within the ranks, but at the same time returns value to those that believe in the emblems since the beginning and have accumulated them

**Comment by pashpasho** [comment]
option A is much better but it costs more for new members to join. Some idea...
1- winner of factions competition determine with factions that feeding their pets more effectively and have healthier pet. 
2- instead of price distribution for top 50 donators of each faction. Design a treasury for each faction so winning price deposited to this treasury that can be used by leader's of each faction(chosen by voting system) to benefit long term and healthy path of each faction .something like government that has treasury and this is a index for wealth of each faction and price of emblems and they can buy NFTs and boosts that sustain overall economy of each faction(very complicated😅) but imagine a faction as state with some authority powers that control overall performance of faction.
So overall stronger factions are expensive to enter but has more benefits and wealthier that can feed theirs pets better and with some technology as NFTs and some monthly boosts to keep their pets healthy and win easier but cheaper factions have less benefits but can work better with younger entries to feed their pets and goes in top.

**Comment by Thebyber** [comment]
I think the system is fine as it is but if I wanted to change it I would stick with option B (it is the simplest option to differentiate the tiers with the same number of players to enable competitions between the tiers)

I would be more concerned about focusing on the factions so that they are more entertaining and not a massive donation day after day to be able to compete in the auctions.

In the end, it is becoming an obligation if you want to opt for NFTs at auctions (and even then it is almost impossible to compete for good NFTs due to their low stock)

Why not add functions to the factions to make a competition between them all in a more entertaining way? Weekly faction challenge: See which faction cuts down the greatest number of trees, see which faction collects the most bananas, see which faction collects the most eggs... etc etc

Why not take advantage of the faction minigames to get amber fossil? You have the minigames to play 5 free attempts and then unlimited if you pay SFL, then add another option which is ranked games with 1-3 attempts where you save your best score. At the end of the week all your best daily scores are added up and if you stay above the top 100 you win some amber fossil (this way you add one more option to earn amber fossil without spending a lot of SFL in donations or a lot of money with the NFTs it asks for bert).

To be able to participate in these rankeds, players must be at least tier 4 of the faction, this prevents bots from participating and encourages fair competition.
